io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,90 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon (Test at every major update)
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # SSH Agent access
+  - --socket=ssh-auth
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=home
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-commands:
+      - |
+        mkdir -p /app/bin/extra
+
+        install -Dm 755 apply_extra --target-directory /app/bin/
+        chmod +x /app/bin/apply_extra
+
+        install -Dm 755 enpass --target-directory /app/bin/
+        chmod +x /app/bin/enpass
+        
+        install -Dm 644 io.enpass.Enpass.metainfo.xml --target-directory /app/share/metainfo
+    sources:
+      - type: script
+        dest-filename: enpass
+        commands:
+          - exec /app/extra/enpass/Enpass
+      - type: extra-data
+        filename: enpass.deb
+        only-arches: [x86_64]
+        url: https://apt.enpass.io/pool/main/e/enpass/enpass_6.8.5.1173_amd64.deb
+        sha256: 5855e617041d73682320f3643eb4136c93eef2beaf3be9d37cbadfc76d719b5b
+        x-checker-data:
+          type: debian-repo
+          package-name: enpass
+          root: https://apt.enpass.io
+          dist: stable
+          component: main
+        size: 39644988
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - |
+            FLATPAK_ID=io.enpass.Enpass
+
+            # Extract upstream package
+            mkdir -p deb-package export/{bin,share/{applications,icons}}
+
+            bsdtar -xvf enpass.deb
+            tar -xzf data.tar.gz -C deb-package
+
+            # Install desktop entry and icons
+            mv deb-package/usr/share/{applications,icons} export/share/
+
+            rename enpass ${FLATPAK_ID} export/share/applications/*.desktop
+            rename enpass ${FLATPAK_ID} export/share/icons/hicolor/*/{apps,status}/*
+
+            sed -i "s/Exec=\/opt\/enpass\/Enpass %U/Exec=\/app\/extra\/enpass\/Enpass %F/g" export/share/applications/${FLATPAK_ID}.desktop
```

Comment:
1. you should use `destkop-file-edit`
2. the executable should NOT be an absolute path. Instead it should match the `command` property, modulo the arguments. `enpass %F`

Same for the next two lines.

Reply:
Thanks for the suggestions @hfiguiere 

- `desktop-file-edit` was failing when trying to build locally on an Arch Linux VM, saying it couldn't be found. Consequently, I went the `sed` route.
- Will `${FLATPAK_DEST}/extra/enpass/Enpass` suffice throughout the manifest?

Reply:
Hey @hfiguiere, the bot has successfully built my manifest, and I've verified I can install and run the Flatpak with `flatpak install --user https://dl.flathub.org/build-repo/9396/io.enpass.Enpass.flatpakref`

I'm still only able to get the Flatpak to work with absolute paths for the exec. Variables such as `FLATPACK_DEST`, `FLATPAK_SYSTEM_DIR`, `XDG_RUNTIME_DIR` as prefixes would not run the app.

In addition, `desktop-file-edit` would not work when trying to build locally (even when verified at /usr/bin/desktop-file-edit)` on my builder.